demo/src/Root.jsx

```diff
@@ -3,11 +3,17 @@ import React from 'react';
 import { BrowserRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import App from '../../src/components/app';
+import NavbarContainer from '../../src/containers/navbar';
+import getUserConfirmation from '../../src/utils/getUserConfirmation';
 
-const Root = ({ store, routes, nav }) => (
+const Root = ({ store, routes, navConfig }) => (
   <Provider store={store}>
-    <BrowserRouter>
-      <App routes={routes} navConfig={nav} />
+    <BrowserRouter getUserConfirmation={getUserConfirmation(store.dispatch)}>
```

Comment:
how does this work? i understand its a modal but wasnt able to find an example of it, or force it to prompt me.